Automattic/wp-calypso
# WooCommerce: Remove state as it's no longer in use

#### Changes proposed in this Pull Request

This PR is a follow-up to #51309 and removes more unused WooCommerce extension state. Most of it looked as if it's used, but it's practically not, so we can remove it. I'm providing some more details in a self-review. 

#### Testing instructions

* Verify removed code isn't used.
* Verify the Woo extension still functions the same way.
* Verify tests pass.
